lib/srv/discovery/discovery_test.go

```diff
@@ -827,7 +827,7 @@ func TestDiscoveryInCloudKube(t *testing.T) {
 					Types:      []string{"gke"},
 					Locations:  []string{"*"},
 					ProjectIDs: []string{"p1"},
-					Tags:       map[string]utils.Strings{"env": {"prod"}},
+					Labels:     map[string]utils.Strings{"env": {"prod"}},
```

Comment:
We need to keep one of the tests that uses Tags to test backwards compatibility.